### DISCUSSION TITLE
Disable pwnedpasswords

### DISCUSSION THREAD

**Discussion by p3lim** [dani-garcia/vaultwarden#3644] [Ideas]
### Subject of the issue

During registration there is an option to "Check known data breaches for this password", which can be disabled by the user.

If however this is enabled, and api.pwnedpasswords.com is unreachable, the registration fails silently.

This will happen in corporate environments with a http proxy.

### Deployment environment

* Vaultwarden version: v1.28.1
* Web-vault version: v2023.3.0b

(everything else should be irrelevant for this issue)

### Steps to reproduce

1. Be behind corporate proxy, api.pwnedpasswords.com is not whitelisted
2. Attempt to create a new account, leaving the defaults
3. Clicking "Create account" silently fails, providing no information to the user what went wrong

### Expected behaviour

At least a warning that the call failed.

Preferably an option (server-side) to disable all external calls, but most importantly for this issue to api.pwnedpasswords.com

### Actual behaviour

No info, clicking "Create account" does nothing.

**Comment by BlackDex** [comment]
This is not something we can fix as this is a web-vault built-in item which we can't toggle on/off at runtime.
I suggest to report this upstream that when the hibp api is unreachable it fails without a warning.

**Comment by 72tlukas72** [comment]
We have same problem - we are working in air gapped network and this option is enable by default. Maybe option to disable via .env file?

**Reply by BlackDex** [reply]
No, we can't change that value at runtime, it's built into the web-vault.
